pkg/scc/scc.go

```diff
@@ -206,6 +333,60 @@ func (builder *Builder) WithFSGroup(fsGroup string) *Builder {
 	return builder
 }
 
+func (builder *Builder) WithFSGroupRange(fsGroupMin, fsGroupMax int64) *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Redefining SecurityContextConstraints %s with"+
+		"fsGroupRange: fsGroupMin: %d, fsGroupMax: %d", builder.Definition.Name, fsGroupMin, fsGroupMax)
+
+	if fsGroupMin > fsGroupMax {
+		glog.V(100).Infof("SecurityContextConstraints 'fsGroupMin' argument can not be greater than fsGroupMax")
+
+		builder.errorMsg = "SecurityContextConstraints 'fsGroupMin' argument can not be greater than fsGroupMax"
+
+		return builder
+	}
+
+	if builder.Definition.FSGroup.Ranges == nil {
+		builder.Definition.FSGroup.Ranges = []securityV1.IDRange{{Min: fsGroupMin, Max: fsGroupMax}}
+
+		return builder
+	}
+
+	builder.Definition.FSGroup.Ranges = append(
+		builder.Definition.FSGroup.Ranges, securityV1.IDRange{Max: fsGroupMax, Min: fsGroupMin})
+
+	return builder
+}
+
+// WithGroups adds groups to SecurityContextConstraints.
+func (builder *Builder) WithGroups(groups []string) *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Redefining SecurityContextConstraints %s with"+
+		"groups: %v", builder.Definition.Name, groups)
+
+	if len(groups) == 0 {
+		glog.V(100).Infof("SecurityContextConstraints 'groups' argument cannot be empty")
+
+		builder.errorMsg = "SecurityContextConstraints 'fsGroupType' cannot be empty string"
+
+		return builder
+	}
+
+	if builder.Definition.Groups == nil {
+		builder.Definition.Groups = groups
+	} else {
```

Comment:
Inconsistent code else. In all other function we use different construction.

Please replace by:
```
if builder.Definition.Groups == nil {
		builder.Definition.Groups = groups

                return builder
}

builder.Definition.Groups = append(builder.Definition.Groups, groups...)

retrun builder 
```